### ISSUE TITLE
Provide minimum webpack@5 example that can build latest SwaggerUI/swagger-ui-react

### ISSUE THREAD

**Issue by char0n** [swagger-api/swagger-ui#7910]
Having minimum webpack@5 config, it can help also people with different build systems like rollup or esbuild.

**Comment by piotrekwitkowski** [comment]
Adding my 0.02$: it's not always the case that devs have access to **any** bundler's config. If modifying config is necessary, for example:
- `swagger-ui-react` won't work for people using `create-react-app` without ejecting (me) and
- `swagger-ui` won't work with `ng build` (see #7902).

Providing users with config copy-paste bits seems bad from the UX perspective.
Instead of writing documentation on how to fix the symptoms, why not remove the cause and make swagger-ui work out-of-the-box without node polyfills that must be configured manually?

**Comment by scottohara** [comment]
Possible duplicate of #7908

**Comment by char0n** [comment]
@piotrekwitkowski you're right. I've issued https://github.com/swagger-api/swagger-ui/pull/7946 that addresses the problem. After merging the PR, SwaggerUI will work out of the box with CRA@{4,5} and should work with ng build as well. Thanks for bringing different way of thinking into this problem.

**Comment by piotrekwitkowski** [comment]
Great news! Looking forward to see this shipped and finally be unblocked! :)

**Comment by char0n** [comment]
Fixed in https://github.com/swagger-api/swagger-ui/releases/tag/v4.10.0
